recipes/bzip2/1.0.8/conanfile.py

```diff
@@ -57,6 +53,6 @@ def package(self):
         cmake.install()
         tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
 
-
     def package_info(self):
-        self.cpp_info.libs = ['bz2']
+        self.cpp_info.name = "BZip2"
+        self.cpp_info.libs = tools.collect_libs(self)
```

Comment:
Question: Didn't we want to avoid tools.collect_libs when possible? 🤔

Reply:
I don't know the answer, but if we do, can we please add it to the wiki and conan-center hook?

Reply:
in general, I'd avoid usage of it. `tools.collect_libs` returns alpha-numeric sorted list of libraries withing `self.cpp_info.libdirs`. it might be undesired in several cases:
1. there are libraries which shouldn't be linked by default (e.g. `preloadable_libiconv.so` in case of `libiconv`
2. there are more than just one library within the directories, and linking order is important (e.g. project is not Windows-only)
in these cases the result of `tools.collect_libs` is not the expected and intended one. for `bzip2` it might be harmless (e.g. there is only one library), but in generic case it causes troubles.

Reply:
Opened an issue about this here https://github.com/conan-io/conan-center-index/issues/302 please comment there